torch/onnx/symbolic_helper.py

```diff
@@ -521,37 +521,63 @@ def _get_dim_for_cross(x: _C.Value, dim: Optional[int]):
     return dim
 
 
-def _unimplemented(op: str, msg: str):
+def _unimplemented(op: str, msg: str, value: Optional[_C.Value] = None) -> None:
     # For BC reasons, the behavior for Caffe2 does not raise exception for unimplemented operators
     if _C_onnx._CAFFE2_ATEN_FALLBACK:
-        warnings.warn(
-            "ONNX export failed on " + op + " because " + msg + " not supported"
-        )
+        warnings.warn(f"ONNX export failed on {op} because {msg} not supported")
     elif GLOBALS.operator_export_type == _C_onnx.OperatorExportTypes.ONNX:
-        _onnx_unsupported(f"{op}, {msg}")
+        _onnx_unsupported(f"{op}, {msg}", value)
 
 
-def _onnx_unsupported(op_name: str):
-    raise RuntimeError(
+def _onnx_unsupported(op_name: str, value: Optional[_C.Value] = None) -> None:
+    message = (
         f"Unsupported: ONNX export of operator {op_name}. "
-        "Please feel free to request support or submit a pull request on PyTorch GitHub."
+        f"Please feel free to request support or submit a pull request "
+        f"on PyTorch GitHub: {_constants.PYTORCH_GITHUB_ISSUES_URL}"
     )
+    if isinstance(value, _C.Value):
+        raise errors.SymbolicValueError(
+            message,
+            value,
+        )
+    raise RuntimeError(message)
 
 
-def _onnx_opset_unsupported(op_name: str, current_opset: int, supported_opset: int):
-    raise RuntimeError(
+def _onnx_opset_unsupported(
+    op_name: str,
+    current_opset: int,
+    supported_opset: int,
+    value: Optional[_C.Value] = None,
+) -> None:
+    message = (
         f"Unsupported: ONNX export of {op_name} in opset {current_opset}. "
         f"Please try opset version {supported_opset}."
     )
+    if isinstance(value, _C.Value):
+        raise errors.SymbolicValueError(
+            message,
+            value,
+        )
+    raise RuntimeError(message)
```

Comment:
```suggestion
    raise UnsupportedOperatorError(message)
```
Why are we keeping all those `RuntimeError`?

Reply:
Good call. I didn't realize we can use them

Reply:
I realized `UnsupportedOperatorError` is semantically different than these functions, where `UnsupportedOperatorError` typically says an op is not supported _at all_ in one version but is in another, but these functions are called when certain arguments not supported (partial support exists).

I suggest keeping this PR focused on enabling the `SymbolicValueError` and retaining the original behavior (RuntimeError) when `SymbolicValueError` cannot be constructed. Then we revisit this to unify the rest of the errors to make them make more sense. What do you think?

Reply:
Changed to OnnxExporterError